packages/contracts/contracts/ChugSplashRegistry.sol

```diff
@@ -53,23 +67,48 @@ contract ChugSplashRegistry {
      */
     mapping(ChugSplashManager => bool) public managers;
 
+    mapping(bytes32 => address) public adapters;
+
     /**
      * @notice Registers a new project.
      *
-     * @param _name  Name of the new ChugSplash project.
-     * @param _owner Initial owner for the new project.
+     * @param _name         Name of the new ChugSplash project.
+     * @param _owner        Initial owner for the new project.
+     * @param _proxyUpdater Address of the ProxyUpdater for this project. Projects that do not use a custom
+     *                      proxy should set this to the zero-address.
+     * @param _proxyAdmin   Address of the ProxyAdmin for this project. Projects that do not use a custom
+     *                      proxy should set this to the zero-address.
      */
-    function register(string memory _name, address _owner) public {
+    function register(
+        string memory _name,
+        address _owner,
+        address _proxyUpdater,
+        address _proxyAdmin
```

Comment:
I think the `ProxyAdmin` contract can just be created within the `register` function to avoid the need for a user to create a second contract.